modules/ossm-cert-manager-istio-csr-revisionbased-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="revision-based-istio-csr-installation_{context}"]
+= istio-csr installation: RevisionBased update strategy
+
+If you plan to set the `spec.updateStrategy` field to `RevisionBased` when you create and install your `Istio` resource, follow this procedure.
+
+.Procedure
+
+. Specify all the Istio revisions to your `istio-csr` deployment.
+
+. Add the jetstack charts to your local Helm repo by running the following command:
++
+[source, terminal]
+----
+helm repo add jetstack https://charts.jetstack.io --force-update
```

Comment:
terminal code blocks must be prefixed with a command prompt. This applies throughout.